scripts/xtensa-build-all.sh

```diff
@@ -79,6 +84,17 @@ then
 	exit 1
 fi
 
+if [ "x$USE_PRIVATE_KEY" != "xyes" ]
+then
+	RIMAGE_PRIVATE_KEY="${pwd}/rimage/keys/otc_private_key.pem"
```

Comment:
I think this can be removed, because it's going to be in cmake file. If there is no `-k` arg just pass nothing to cmake and it will use default for both `scripts` users and ppl that build FW without `scripts`.

Reply:
ok.